mlir/lib/Conversion/AIRToAIEPass.cpp

```diff
@@ -51,6 +51,20 @@ struct AIRToAIEOptions {
   bool emit_herd_lock;
 };
 
+std::vector<int> l2_dma_cols{7, 8, 9, 10};
+const int l2_dma_channels = 2;
+
+// std::vector<int> s80_nmu_col_list{0, 0, 1, 1, 0, 0, 1, 1,
+//                                   0, 0, 1, 1, 0, 0, 0, 0,
+//                                   0, 0, 1, 1, 0, 0, 0, 0,
+//                                   0, 0, 1, 1, 0, 0, 0, 0,
+//                                   0, 0, 1, 1, 0, 0, 0, 0,
+//                                   0, 0, 1, 1, 0, 0, 1, 1,
+//                                   0, 0};
+std::vector<int> shim_dma_cols{2,  3,  6,  7,  10, 11, 18, 19,
+                               26, 27, 34, 35, 42, 43, 46, 47};
+const int shim_dma_channels = 2;
```

Comment:
Not necessarily for this PR but it would be a good idea to start factoring device/generation specific information out of passes

Reply:
Anything related to L2 can be deleted now. The shim tile information is needed until we get some kind of logical tile in aie dialect.

Reply:
There's two L2 tests for AIRToAIE in `mlir/test/Conversion/AIRToAIE/`, air_aie_to_L2cpy.mlir and air_L2cpy_to_aie.mlir, should I remove those as well or disable them with a 'requires' statement?

Reply:
They can be removed

Reply:
Yes that would be better, I'll adjust that. Thank you!